onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -470,6 +473,7 @@ def quantize_static(
         ("CalibMovingAverage", "moving_average"),
         ("CalibMovingAverageConstant", "averaging_constant"),
         ("CalibMaxIntermediateOutputs", "max_intermediate_outputs"),
+        ("CalibStridedMinMax", "stride"),
```

Comment:
Is the stride input provided via a parameter to `quantize_static` or via `extra_options`? I see some code for both.

Reply:
I would probably recommend using the extra_option path only.